Makefile

```diff
@@ -245,12 +245,12 @@ test-unit-local: bin/skopeo
 	$(GO) test $(MOD_VENDOR) -tags "$(BUILDTAGS)" $$($(GO) list $(MOD_VENDOR) -tags "$(BUILDTAGS)" -e ./... | grep -v '^github\.com/containers/skopeo/\(integration\|vendor/.*\)$$')
 
 vendor:
-	$(GO) mod tidy
+	$(GO) mod tidy -compat=1.17
```

Comment:
I’m not too happy with this; it’s going to stay around forgotten forever.

AFAICT the typical effect of keeping compatibility with 1.16 is
- to have to copy&paste a command from an error message, and run it — sure, a hassle
- to end up with newer versions of some dependencies — a good thing.